### ISSUE TITLE
Feature Request: Allow order of MFA options to be configurable

### ISSUE THREAD

**Issue by greyknight79** [okta/okta-signin-widget#538]
### I'm submitting a
  - [ ] bug report
  - [x] feature request

### Background info

When a user logs in and needs to setup an MFA option, allow the list of choices to be ordered through configuration.

**Comment by clifrender** [comment]
Yes, please. This would be great. Our marketing department won't let us use certain options because they want to be able to control the order in which they appear in the list.
